tests/selenium/inline_msg.py

```diff
@@ -16,17 +18,21 @@ def set_inline_message_test(self):
         self.dropdown_test('email_setting', 'all_email_since', '-1 week', '-5 years')
 
     def navigate_msg_test(self):
-        self.by_css('[data-source=".email_folders"]').click()
-        allmsgs = self.by_class('menu_email')
-        allmsgs.find_element_by_tag_name('a').click()
-        self.wait_on_class('checkbox_cell')
-        body = self.by_class('message_table_body')
-        subject = body.find_element_by_class_name('subject')
-        subject.find_element_by_tag_name('a').click()
-        self.wait_on_class('header_subject')
-        detail_subject = self.by_class('header_subject')
-        header = detail_subject.find_element_by_tag_name('th')
-        assert header.text.startswith('recent')
+        try:
+            self.by_css('[data-source=".email_folders"]').click()
+        except NoSuchElementException:
+            pass
```

Comment:
Seems incorrect - why do we have a test that tries to check for something but pass if it is not found?

Reply:
I need an idea about this, This menu is now displaying only if we have more that one server(servers+feeds). and in my case i'm trying to access without even one connected server.

Reply:
Then skip the test (check) unless you are sure there is a server or feed - e.g. check this only after you add one server (there is code to do this in the tests, right?)